docs/faq.md

```diff
@@ -158,15 +160,15 @@ Chromebooks will need higher requirements if you're using Linux (Beta)/Crostini.
 These specs will provide full-speed gameplay almost everywhere, with some slowdowns in non-critial areas with a large amount of objects, such as the main caverns of Area 2, Area 3, and Area 5. Weaker GPUs may slow down in Fusion difficulty during X Parasite morphing.
 
 #### Where can I stay up-to-date on AM2R?  
-<https://bit.ly/AM2Rblog>  
-<https://reddit.com/r/AM2R>  
-<https://discord.gg/YTQnkAJ>  
-<https://youtube.com/c/am2rcommunityupdates>
+- [Reddit](https://reddit.com/r/AM2R) <br> <!--<br> needed for formatting-->
+- [Discord](https://discord.gg/YTQnkAJ) and [Matrix](https://matrix.to/#/#am2r:matrix.org) (Connected via a cross-platform bridge) <br>
+- [Community Updates Youtube Channel](https://youtube.com/c/am2rcommunityupdates) <br>
+- [Legacy AM2R blog](https://metroid2remake.blogspot.com/) (No community updates will be present here)
```

Comment:
I added the `legacy` tag as it may confuse users who try to find the community updates, then only see the DMCA/Slippy. `Doctor M64's AM2R blog` is ok, as long as it is clarified to not be a place to find community updates.

Reply:
Some more thoughts:
- would be good to bring the bit.ly link back, as it probably gives doc a few bucks here and there
- Docs link was at the very top as a thing out of respect, since he was (one of) the original creators for AM2R
- Clarification can probably be done via a `-` between link and description. Can then also be done for the other lists as consistency, since most of them would also benefit from a small description.

Reply:
DoctorM64's AM2R blog - Archive of the original development directly from the original creator. **No community updates or support will be found here**

Reddit - Subreddit of AM2R discussion, support can be found here, but mainly gameplay videos and news are posted.

Discord/Matrix - Server of AM2R, will help with installation and discussion of AM2R. Discord and Matrix are bridged together.

Community Updates YouTube - Video tutorials, announcements, and gameplay are shown in the updates YouTube channel

Reply:
> Archive of the original development directly from the original creator. No community updates or support will be found here

Several things I don't like here:
- It's not really an archive per se
- DoctorM64 is not the sole original creator, but one of many
- The boldness of the last sentence
- Currently makes it seem like there's not really much of importance there

I'd also significantly shorten the descriptions:

```diff
- Subreddit of AM2R discussion, support can be found here, but mainly gameplay videos and news are posted.
+ The official AM2R Subreddit.
```

```diff
- Discord/Matrix - Server of AM2R, will help with installation and discussion of AM2R. Discord and Matrix are bridged together.
+ Discord/Matrix - The official AM2R Server/Space, both of which are bridged together.
```

```diff
- Community Updates YouTube - Video tutorials, announcements, and gameplay are shown in the updates YouTube channel
+ Community Updates YouTube - The official AM2R Community Updates YouTube channel.
```

Reply:
Oh, also I double checked, and turns out that the bitly doesn't make money. So feel free to revert that to the original link in order to avoid a redirect.

Reply:
Your descriptions are helpful for the communication platforms, I will update them. On the topic of the AM2R, I agree that it is not an archive and still updated. On the other hand however, users who go to here will not get any help for community updates and may spam the comments of his recent slippy post. I agree that we should honor the original AM2R team for their work, but it can't be ignored that users will receive no help from the blog, and may even confuse skippy for AM2R community updates. How does these options sound? 
`Original AM2R blog by DoctorM64 and AM2R development team, not associated with current development`
or
`Original AM2R blog by DoctorM64 and AM2R development team, no downloads for AM2R launcher due to DMCA`

I feel they credit all individuals of the AM2R development team while clarifying they are not apart of current development, but there is room for improvement.

Reply:
Am more a fan of the first one, would change it slightly to `Original AM2R blog by DoctorM64. DoctorM64 and the rest of the  original AM2R development team have no association with the Community Updates`.

Reply:
Moved to the top and added the updates, formatting does not appear to brake, and the links all seem to work out now. i'll keep this open if there is anything else you want to change.

Reply:
Looks fine by me, feel free to resolve.